tests/lib/openstack_query/test_query_executer.py

```diff
@@ -0,0 +1,141 @@
+from unittest.mock import MagicMock, patch
+import pytest
+from openstack_query.query_executer import QueryExecuter
+from tests.lib.openstack_query.mocks.mocked_props import MockProperties
+
+# pylint:disable=protected-access
```

Comment:
Can we drop this? I'm really not keen on protected-access as it's usually a sign the class is doing too much, since we need to observer intermediate state

Reply:
How would you test whether a setter works then?

Reply:
Just exercise both at the same time, if you set a value do you get it back
The setter getter should be doing validation at most (e.g. `isinstance(list)`), not manipulating the value

If we need to manipulate the value in a complex way then it's maybe a sign the value should be part of a class